<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>new_save</class>
 <widget class="QDialog" name="new_save">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>298</width>
    <height>111</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Save</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="name_la">
       <property name="text">
        <string>Enter a project name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="name_le"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="save_buttion">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancel_buttion">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
